File: src/main/java/org/dedda/bratwurst/parse/AssertFalseParser.java
package org.dedda.bratwurst.parse;

import org.dedda.bratwurst.lang.assertions.AssertEquals;
import org.dedda.bratwurst.lang.assertions.AssertFalse;
import org.dedda.bratwurst.lang.assertions.AssertTrue;

/**
 * Created by dedda on 1/23/16.
 *
 * @author dedda
 */
public class AssertFalseParser {

    public AssertFalse parse(String line, int lineNumber) {
        String varName = line.substring(3, line.length() - 2);
        return new AssertFalse(lineNumber, varName);
    }

}

File: src/main/java/org/dedda/bratwurst/parse/AssertFalseParser.kt
package org.dedda.bratwurst.parse

import org.dedda.bratwurst.lang.assertions.AssertFalse

/**
 * Created by dedda on 1/23/16.
 *
 * @author dedda
 */
class AssertFalseParser {
    fun parse(line: String, lineNumber: Int): AssertFalse {
        val varName = line.substring(3, line.length - 2)
        return AssertFalse(lineNumber, varName)
    }
}